src/command/dev/next/mod.rs

```diff
@@ -63,51 +71,61 @@ impl Dev {
         let graph_ref = self.opts.supergraph_opts.graph_ref.clone();
         let composition_output = tmp_config_dir_path.join("supergraph.graphql");
 
-        let one_off_composition = OneShotComposition::builder()
-            .client_config(client_config)
-            .profile(profile)
-            .elv2_license_accepter(self.opts.plugin_opts.elv2_license_accepter)
-            .skip_update(self.opts.plugin_opts.skip_update)
+        let one_shot_composition = OneShotComposition::builder()
+            .client_config(client_config.clone())
+            .profile(profile.clone())
+            .elv2_license_accepter(elv2_license_accepter)
+            .skip_update(skip_update)
             .output_file(composition_output)
             .and_federation_version(federation_version)
-            .and_graph_ref(graph_ref)
+            .and_graph_ref(graph_ref.clone())
             .and_supergraph_yaml(supergraph_yaml)
-            .and_override_install_path(override_install_path)
+            .and_override_install_path(override_install_path.clone())
             .build();
 
-        // FIXME: send this off to the router binary
-        let _composition_output = one_off_composition.compose().await?;
+        // The router binary will know where to find the composition result; we compose initially
+        // for the router to have a properly composed schema when starting
+        one_shot_composition.compose().await?;
```

Comment:
Should we have this produce the file that composition is written to, just to make it clear?

Reply:
will add a todo! you want the filepath that gets passed on? right now, we just find it by the temp dir